### ISSUE TITLE
RFE: systemd-nspawn: --boot=/path/to/command and Boot=/path/to/command

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [systemd/systemd#12703]
This request arises from the need to provide consistent/symmetric patterns 
for invocation of `systemd-nspawn` via both command line and with `*.nspawn` profile.

1. currently `systemd-nspawn`  invocation via command line:
https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html
supports `COMMAND` and `--boot` inconsistently:
```
systemd-nspawn [OPTIONS...] [COMMAND [ARGS...] ]
systemd-nspawn --boot [OPTIONS...] [ARGS...]
```
2. and  mirroring `systemd-nspawn` invocation with `*.nspaw` profile:
https://www.freedesktop.org/software/systemd/man/systemd.nspawn.html
has no option corresponding to the `COMMAND`

3. this is a request to add consistent support for both `boot` and `command` as follows:
(also deals with "space separation" problem of `Parameters=`)
a) `systemd-nspawn`:
https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html
`--boot=/path/to/command`
`--parameter=ARG1`
`--parameter=ARG2`
b) `systemd.nspawn`:
https://www.freedesktop.org/software/systemd/man/systemd.nspawn.html
`Boot=/path/to/command`
`Parameter=ARG1`
`Parameter=ARG2`

in other words, both `--boot=` and `Boot=` apart form `yes` and `no` should support an absolute path expression which must point to the executable in the machine image

**Comment by keszybz** [comment]
Everything is already possible, but the documentation wasn't entirely clear. I'll push a small patch to enhance the docs. Please take a look, it should answer all the questions here.
